discovery-provider/src/queries/get_health.py

```diff
@@ -59,6 +59,28 @@ def get_elapsed_time_redis(redis, redis_key):
     return elapsed_time_in_sec
 
 
+def get_elapsed_time_postgres(db, tablename):
```

Comment:
Not sure if these functions belong in get_health if they're not used in the health check. Maybe the IndexingCheckpoints query can be grouped here: https://github.com/AudiusProject/audius-protocol/blob/89cc1acaa988d9653cd2991a4448a50743d97281/discovery-provider/src/utils/update_indexing_checkpoints.py.